.circleci/config.yml

```diff
@@ -2,6 +2,8 @@ defaults: &defaults
   docker:
     - image: 'circleci/node:12'
   working_directory: ~/project/vaken
+  environment:
+    CC_TEST_REPORTER_ID: 'b6cea094202296cdea6867cd99c1efa9aab7ad0a9679ec999e890d3fd3bcaaf0' # write only so all good
```

Comment:
Could you elaborate on "write only so all good"? I get spooked when I see tokens in git

Reply:
It lets you write to our codeclimate instance but you have to be using a similar looking (unclear how codeclimate decides that) repo to do that. I'll just add it as an env var - should be easy enough.